code/addons/docs/src/preset.ts

```diff
@@ -61,15 +63,22 @@ async function webpack(
     mdxBabelOptions,
     configureJSX = true,
     csfPluginOptions = {},
+    mdxPluginOptions = {},
     sourceLoaderOptions = null,
     transcludeMarkdown = false,
   } = options;
 
-  const mdxLoaderOptions = await options.presets.apply('mdxLoaderOptions', {
+  const mdxLoaderOptions: CompileOptions = await options.presets.apply('mdxLoaderOptions', {
     skipCsf: true,
+    ...mdxPluginOptions,
```

Comment:
@shilman How do you think we should merge these configs?

This setup always respects skipCsf but merges everything else